scripts/k8s-storage/driver-cephfs.yaml

```diff
@@ -0,0 +1,67 @@
+---
+ShortName: cephcsi-cephfs-test
+StorageClass:
+  FromExistingClassName: k8s-storage-e2e-cephfs
+  # FromFile: sc-cephfs.yaml
+
+SnapshotClass:
+  # Must be set to enable snapshotting tests
+  FromName: false
+
+DriverInfo:
+  # Internal name of the driver, display name in the test case and test objects
+  Name: test.cephfs.csi.ceph.com
```

Comment:
as this is for logging for better clarity just call it as with cephfs.csi.ceph.com? same for rbd also

Reply:
sure, I can do that